src/org/mozilla/javascript/NativeArray.java

```diff
@@ -441,6 +445,9 @@ public Object execIdCall(
                 case Id_flat:
                     return js_flat(cx, scope, thisObj, args);
 
+                case Id_flat_map:
+                    return js_flat_map(cx, scope, thisObj, args);
```

Comment:
```suggestion
                case Id_flatMap:
                    return js_flatMap(cx, scope, thisObj, args);
```
I want the case to be the same as other method names.
https://github.com/mozilla/rhino/blob/90d3b44e91fd70e7291dc9572eacc0376afa7872/src/org/mozilla/javascript/NativeArray.java#L430-L431